unittests/tools/test_nuclei_parser.py

```diff
@@ -151,6 +151,12 @@ def test_parse_many_findings_new(self):
             self.assertEqual("nuclei-example.com", finding.unsaved_endpoints[0].host)
             self.assertEqual(22, finding.unsaved_endpoints[0].port)
             self.assertEqual("CVE-2018-15473", finding.vuln_id_from_tool)
+            vulnerability_ids = finding.unsaved_vulnerability_ids
+            self.assertEqual(1, len(vulnerability_ids))
+            self.assertEqual('cve-2018-15473', vulnerability_ids[0])
```

Comment:
`cve` should be in uppercase, e.g.  `CVE-2018-15473`, otherwise the link in the UI might not work.